### ISSUE TITLE
Visual Studio 2017 RC C# Notepad Sample Does Not Work for FindElementByAccessibilityId

### ISSUE THREAD

**Issue by ianceicys** [Microsoft/WinAppDriver#89]
I tried out the latest build of WinAppDriver and the Notepad Sample and couldn't get the test to run successfully due the file path edit box not working.

https://github.com/Microsoft/WinAppDriver/blob/master/Samples/C%23/NotepadTest/Scenario.cs#L63

https://github.com/Microsoft/WinAppDriver/tree/master/Samples/C%23/NotepadTest

Here is the video of my repo:

https://www.youtube.com/watch?v=itbGurSPLiA

WinVer: Windows 10 Build 1511 (OS Build 10586.633)
WinAppDriver - 0.6.1610.28001
Microsoft Visual Studio Enterprise 2017 RC
Version 15.0.25914.0 D15REL

**Comment by yodurr** [comment]
Hi Ian,

VS 2017 doesn't appear to be the issue, and WAD 0.6 seems ok as well.
The issue I'm seeing looks related to behavior on that version of Windows.  For me the failure is on line 109.  To get it passing I removed the file extension from the **TestFileName** global so it's just "NotepadTestOutputFile".  Now the test passes but the file explorer window isn't getting closed.

What I saw in your video was different with the text getting entered into the filename box, and I haven't been able to reproduce this.  Is the file name being typed into Notepad a consistent repro?  Is it possible something caused focus to move back to the notepad window during the test?

Thanks.

**Comment by yodurr** [comment]
please open new issue if there's still an issue with the notepad test.  Thank you.

**Comment by timotiusmargo** [comment]
Hi @ianceicys,

We have just updated our C# samples in the repository. The [NotepadTest](https://github.com/Microsoft/WinAppDriver/tree/master/Samples/C%23/NotepadTest) you mentioned above should no longer have the issue. They are tested on Windows 10 PC with the latest Windows 10 version (Version 1607 or later). Please give it a try!
